compact/definitions.xml

```diff
@@ -410,6 +410,50 @@ Examples:
     <constant name="SolenoidBarrel_rmin"       value="Solenoid_rmin"/>
     <constant name="SolenoidBarrel_length"     value="Solenoid_length"/>
 
+    <documentation>
+      ## IP6 Beam Pipe Parameters - Note might need cleaning
+    </documentation>
+    <constant name="CrossingAngle" value="-0.025*rad "/>
+    <constant name="ionCrossingAngle" value="CrossingAngle"/>
+    <constant name="electronCrossingAngle" value="0.0"/>
+    <constant name="CrossingSlope" value="CrossingAngle"/>
```

Comment:
Is this used, and do we want to use "slope" which has vertical angle implications? In my interpretation, slope is zero at the IP.

Reply:
I'll open a separate issue where these kind of naming comments, unused and copied variables can be highlighted. If we started digging into this here it could take a while.